Formula/b/bzt.rb

```diff
@@ -204,8 +203,8 @@ class Bzt < Formula
   end
 
   resource "setuptools" do
-    url "https://files.pythonhosted.org/packages/5e/11/487b18cc768e2ae25a919f230417983c8d5afa1b6ee0abd8b6db0b89fa1d/setuptools-72.1.0.tar.gz"
-    sha256 "8d243eff56d095e5817f796ede6ae32941278f542e0f941867cc05ae52b162ec"
+    url "https://files.pythonhosted.org/packages/c5/41/247814d8b7a044717164c74080725a6c8f3d2b5fc82b34bd825b617df663/setuptools-65.5.0.tar.gz"
```

Comment:
only bzt has setuptools dep
```
$ pipgrip --tree bzt
bzt (1.16.33)
...
├── setuptools==65.5.0 (65.5.0)
```

Reply:
In https://github.com/Homebrew/homebrew-core/pull/181446 I moved setuptools to a regular formula runtime dependency, because while it seems we can use a new setuptools just fine, it seemed to me using it as a `resource` block would cause friction during autobumps.

Reply:
(nvm, that failed on CI, going to instead try with manual bump up of the resource block to current version)